### PR TITLE
Update foodcritic docs site per v13.1.0 & v14.0.0

### PR DESCRIPTION
**Pull request by jeffbyrnes**
The docs site has lagged behind reality. This updates things as of [foodcritic v14.0.0](https://github.com/Foodcritic/foodcritic/blob/master/CHANGELOG.md#1400-2018-06-28)

### CODE REVIEW

File: rules.yml [hunk 1]
```diff
@@ -2361,3 +2364,21 @@ rules:
...
+    tags:
+      - correctness
+    summary: |
+      A resource's name should not be set via the `name` property.
+  - code: FC121
+    name: Cookbook depends on cookbook made obsolete by Chef 14
+    tags:
+      - correctness
+    summary: |
+      The `build-essential`, `dmg`, `chef_handler`, `chef_hostname`, `mac_os_x`, `swap`, or `sysctl` cookbooks have been deprecated by built-in resources included with Chef 14.
```
Comment (tas50) [inline rules.yml]: We should probably mention that removing these dependencies also requires the user bump the chef_version metadata to >= 14.0

**Comment by tas50** [comment]
Oh wow thanks for doing this. It's been on my whiteboards and I've been slacking.

File: rules.yml (hunk 1 above)
Comment (jeffbyrnes) [reply rules.yml]: @tas50 updated & rebased!

**Comment by jeffbyrnes** [comment]
@tas50 you’re welcome! I found myself trying to reference these rules in Dash, and finding them missing. Checked the website… also missing! So here we are. Glad I could help :)

**Comment by tas50** [comment]
Thanks

**Pull request merged**